packages/trace-viewer/src/sw/snapshotRenderer.ts

```diff
@@ -427,25 +427,26 @@ function snapshotScript(...targetIds: (string | undefined)[]) {
           for (const canvas of canvasElements) {
             const context = canvas.getContext('2d')!;
 
-            const boundingRect = canvas.getBoundingClientRect();
-            const xStart = boundingRect.left / window.innerWidth;
-            const yStart = boundingRect.top / window.innerHeight;
-            const xEnd = boundingRect.right / window.innerWidth;
-            const yEnd = boundingRect.bottom / window.innerHeight;
-
-            const partiallyUncaptured = xEnd > 1 || yEnd > 1;
-            const fullyUncaptured = xStart > 1 || yStart > 1;
+            drawCheckerboard(context, canvas);
```

Comment:
Let's keep not drawing the checkerboard for `fullyUncaptured` canvases, to avoid any performance penalties.